core/src/mender-client.c

```diff
@@ -905,10 +905,16 @@ mender_client_update_work_function(void) {
                     goto END;
                 }
 
-                mender_log_info("Downloading deployment artifact with id '%s', artifact name '%s' and uri '%s'",
-                                deployment->id,
-                                deployment->artifact_name,
-                                deployment->uri);
+#if CONFIG_MENDER_LOG_LEVEL >= MENDER_LOG_LEVEL_INF
+                char id_short[7];
+                if (strlen(deployment->id) >= 7) {
+                    strncpy(id_short, deployment->id, 7);
```

Comment:
I'd use `sizeof(id_short)` here and above so that we can simply update a single place when we decide we want this longer.